azext_iot/iothub/_help.py

```diff
@@ -316,6 +316,113 @@ def load_iothub_help():
+                       device twins, module identities and module twins).
+
+                       For imported endpoints with system assigned identity authentication, the specified hub must have
+                       the correct permissions. Otherwise the command will fail.
+
+                       Private endpoints will be ignored in the import process.
+
+                       For more information, see aka.ms/iothubstate
+        examples:
+        - name: Import the supported state from the specified file to the specified hub.
+          text: >
+            az iot hub state import -n {iothub_name} -f {state_filename}
+        - name: Import the supported state from the specified file to the specified hub, overwriting the previous state of the hub. All
+                certificates, configurations, and devices will be deleted before the new state is uploaded.
+          text: >
+            az iot hub state import -n {iothub_name} -f {state_filename} -r
+        - name: Import only the arm template from the specified file to the specified hub. Note that this will create a new hub if
+                it does not exist. The file may contain the devices and configurations but those will be ignored.
+          text: >
+            az iot hub state import -n {iothub_name} -f {state_filename} --aspects arm
+        - name: Import only the devices and configurations from the specified file to the specified hub. Note that this will NOT
+                create a new hub if it does not exist and the command will fail. The file may contain the arm template but that
+                will be ignored.
+          text: >
+            az iot hub state import -n {iothub_name} -f {state_filename} --aspects devices configurations
+    """
+
+    helps[
+        "iot hub state migrate"
+    ] = """
+        type: command
+        short-summary: Migrate the state of one hub to another hub without saving to a file.
+        long-summary: |
+                       If the arm aspect is specified, the hub will be created if it does not exist.
+
+                       The migrated state will include: arm template for hub, hub configurations (including ADM
+                       configurations and edge deployments), device information (including device identites,
+                       device twins, module identities and module twins).
+
+                       For migrated endpoints with system assigned identity authentication, the specified hub must have
+                       the correct permissions. Otherwise the command will fail.
+
+                       Private endpoints will be ignored in the migration process.
+
+                       If you have trouble migrating, please use the export and import commands to have a file as a backup.
+
+                       For more information, see aka.ms/iothubstate
+        examples:
+        - name: Migrate the supported state of the original hub to the destination hub.
```

Comment:
We use the term `--origin-hub` as the command parameter but in the first 2 example descriptions its "original hub" and the last two its "origin hub"

Reply:
words are hard. English is hard